### PR TITLE
This change is to allow the serve port to be tweaked.

### PR DESCRIPTION
**Pull request by npalumbo**
For example, should we want to clone https://goclone.io and serve on port 8080: 
```
goclone -s -P 8080 https://www.goclone.io/
```

### CODE REVIEW

File: cmd/clone.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"net/http"
 	"net/http/cookiejar"
 	"net/url"
+	"strconv"
```
Comment (mesaglio) [inline cmd/clone.go]:
```suggestion
```

File: cmd/clone.go [hunk 2]
```diff
@@ -74,11 +75,11 @@ func cloneSite(ctx context.Context, args []string) error {
 
 	}
 	if Serve {
-		cmd := exec.Command("open", "http://localhost:5000")
+		cmd := exec.Command("open", "http://localhost:"+strconv.Itoa(ServePort))
```
Comment (mesaglio) [inline cmd/clone.go]:
```suggestion
		serverUrl := fmt.Sprintf("http://localhost:%d", ServePort)
		cmd := exec.Command("open", serverUrl)
```

File: pkg/server/server.go [hunk 3]
```diff
@@ -3,10 +3,11 @@ package server
 import (
 	"github.com/labstack/echo"
 	"github.com/labstack/echo/middleware"
+	"strconv"
```
Comment (mesaglio) [inline pkg/server/server.go]:
```suggestion
	"fmt"
```

File: pkg/server/server.go [hunk 4]
```diff
@@ -20,5 +21,5 @@ func Serve(projectPath string) error {
 	e.Static("/", projectPath)
 	e.File("/", projectPath)
 
-	return e.Start(":5000")
+	return e.Start(":" + strconv.Itoa(port))
```
Comment (mesaglio) [inline pkg/server/server.go]:
```suggestion
	return e.Start(fmt.Sprintf(":%d", port))
```

**Comment by mesaglio** [comment]
This feature looks good, check the changes I suggest and pls resolve de confilcts.

File: cmd/clone.go (hunk 1 above)
Comment (npalumbo) [reply cmd/clone.go]: adopted

File: cmd/clone.go (hunk 2 above)
Comment (npalumbo) [reply cmd/clone.go]: adopted suggestion

File: pkg/server/server.go (hunk 4 above)
Comment (npalumbo) [reply pkg/server/server.go]: adopted suggestion

File: pkg/server/server.go (hunk 3 above)
Comment (npalumbo) [reply pkg/server/server.go]: adopted suggestion

**Comment by npalumbo** [comment]
> This feature looks good, check the changes I suggest and pls resolve de confilcts.

Thanks. I adopted the suggested changes and resolved the conflicts.

**Comment by mesaglio** [comment]
LGTM
We can make a new release with this feature no? @imthaghost

**Review by mesaglio — APPROVED** [review]

**Pull request merged**